src/components/Accordion/__snapshots__/AccordionItem.story.storyshot

```diff
@@ -44,12 +44,17 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT/Accor
               height={16}
               preserveAspectRatio="xMidYMid meet"
               role="img"
+              style={
+                Object {
+                  "willChange": "transform",
+                }
+              }
               viewBox="0 0 16 16"
               width={16}
               xmlns="http://www.w3.org/2000/svg"
             >
-              <polygon
-                points="11,8 6,13 5.3,12.3 9.6,8 5.3,3.7 6,3"
+              <path
+                d="M11 8L6 13 5.3 12.3 9.6 8 5.3 3.7 6 3z"
```

Comment:
I think this is an invalid snapshot generated with an outdated version of Carbon. We recently upgraded Carbon to v10.10.3 in #1095 

To fix, pull in latest from `master`, run `yarn install && yarn test:update`. The polygon is valid - style willChange transform above should be removed. Looks like this entire snapshot change should be effectively discarded.

Reply:
This is due to the component being changed from `AccordionItem` to `AccordionItemDefer`. This file should just be deleted

Reply:
I have deleted AccordionItem storyshot.